### PR TITLE
Completed prep-javascript-functions.

### PR DESCRIPTION
**Pull request by m-Cho1**
prep-javascript-functions

### CODE REVIEW

File: prep-javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+function getCircleCircumference (radius) {
+  return 2*(Math.PI)*radius;
+}
+console.log('getCircleCircumference Exercise:', getCircleCircumference(5));
+
+function getFullName (firstName, lastName) {
+  firstName="Juan"
+  lastName="Ramirez"
+  return firstName + ' '+ lastName;
+}
```
Comment (thebearingedge) [inline prep-javascript-functions/main.js:47]: For this function we can concatenate the parameters.

```js
function getFullName (firstName, lastName) {
  return firstName + ' '+ lastName;
}
```

File: prep-javascript-functions/main.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+console.log('getFullName Exercise:', getFullName());
+
```
Comment (thebearingedge) [inline prep-javascript-functions/main.js:49]: When you call the function you can pass any first name and last name

```js
console.log('getFullName Exercise:', getFullName('Minji', 'Cho'));

```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by m-Cho1 — COMMENTED** [review]
image

File: prep-javascript-functions/main.js (hunk 2 above)
Comment (m-Cho1) [reply prep-javascript-functions/main.js]: that makes more sense. Thank you!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**